### PR TITLE
[file] Update description of state

### PR DESCRIPTION
**Pull request by ad-m**
+label: docsite_pr

##### SUMMARY

Not applicable.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

module file

##### ANSIBLE VERSION

Not applicable.

##### ADDITIONAL INFORMATION

Not applicable.

### CODE REVIEW

File: lib/ansible/modules/files/file.py [hunk 1]
```diff
@@ -40,9 +40,9 @@
     description:
       - If C(directory), all immediate subdirectories will be created if they
         do not exist, since 1.7 they will be created with the supplied permissions.
-        If C(file), the file will NOT be created if it does not exist, see the M(copy)
-        or M(template) module if you want that behavior.  If C(link), the symbolic
-        link will be created or changed. Use C(hard) for hardlinks. If C(absent),
+        If C(file), the file will NOT be created if it does not exist, see the C(touch)
```
Comment (dharmabumstead) [inline lib/ansible/modules/files/file.py]: Please either change the comma to a semicolon here. Thanks!

**Review by dharmabumstead — CHANGES_REQUESTED** [review]
Thanks @ad-m! Please see the review comment then we're good to merge.

**Comment by ad-m** [comment]
@dharmabumstead fixed.

**Comment by abadger** [comment]
Merged and cherry-picked to stable-2.5 for the 2.5.2 release.

**Pull request merged**